gpiozero/spi_devices.py

```diff
@@ -92,9 +131,9 @@ def _read(self):
     def value(self):
         """
         The current value read from the device, scaled to a value between 0 and
-        1.
+        1 (or -1 to +1 for certain devices operating in differential mode).
```

Comment:
Possibly ... At the moment I don't know whether the MCP ADCs are representative of ADCs generally and thus whether differential can be a simple bool or whether we might need something more complex in future. Hence, for now I've left this as "for future consideration".